### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by hopeforkoami**
This is the final Capstone project, in this project I fetch API data from the [Phone details API](https://phone-specs-api.azharimm.dev/) using Redux Toolkit, UI is built with React. Tests are written using Jest and React Testing Library.
In this branch I:

  - Fetch phone brands' data from the [API](https://phone-specs-api.azharimm.dev/)
  - Fetch brand's phones from the [API](https://phone-specs-api.azharimm.dev/)
  - Store API data using Redux Toolkit
  - Add filtering for brands
  - Write the tests
  - Deploy 
  - My pull request targets the dev branch of this [project](https://github.com/hopeforkoami/phoneDetailsMaster).
  
  
The demo video of the project can be found in the link below:
[See here](https://drive.google.com/file/d/1adBIgTWWxY36oFl3XamZKJhadqNJ3bnR/view?usp=sharing)

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# phoneDetailsMaster
+Phone Details Master is a single page  web app that fetch the phone-specs-api
```
Comment (fobadara) [inline README.old.md]: [REQUIRED] Kindly delete this Readme file. It is not serving any purpose

File: README.md [hunk 2]
```diff
@@ -1,2 +1,221 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+- [Live Demo Link](http://pdm.nogdevhouse.com/)
+- [Demo video](https://drive.google.com/file/d/1adBIgTWWxY36oFl3XamZKJhadqNJ3bnR/view?usp=sharing)
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly record a video on loom or through any other tool that allows video play online. Videos saved on Google drive require extra setup and tools in order to be viewed

File: README.md [hunk 3]
```diff
@@ -1,2 +1,221 @@
...
+
+```
+  npm test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
```
Comment (fobadara) [inline README.md]: [REQUIRED] Kindly make this singular since you are the only author

File: src/pages/Home.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          <PhoneBrand
+          id={brandData.id}
+          key={brandData.id}
+          brand={brandData.name}
+          total={brandData.nbrePhone}
+          brandQuery={brandData.brandQuery}/>))
+        }
+
+    </ul>
+  </div>);
```
Comment (fobadara) [inline src/pages/Home.js]: [REQUIRED] Kindly ensure that your design follows the template as much as possible. For example in the template, there is no space between the elements and on the sides. But you have spaces there.
image

[REQUIRED]You also don't have the top section present in the template.


image

[REQUIRED] Kindly ensure that the texts and icons are positioned just like it is in the template

File: src/pages/phonesList.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+          <Phone
+          id={phoneData.id}
+          key={phoneData.id}
+          name={phoneData.name}
+          image={phoneData.image} />))
+        }
+
+    </ul>
+  </div>);
+};
```
Comment (fobadara) [inline src/pages/phonesList.js]: [REQUIRED] Kindly ensure that the details page is arranged horizontally just like the one in the template with a top section containing your desired image. 
image

File: src/components/Phone.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  const {
+    name, image, brandQuery,
+  } = props;
+  return (
+    <li className="phone-container">
+    <img src={image} alt="Phone image" className='phoneimage' />
+    <h2 className="phone-name">{name}</h2>
+    <Link to={`/details/${brandQuery}`}>
+      <button type="button" className="show-icon">
+        <img src={ShowIcon} alt="Show Icon" />
```
Comment (fobadara) [inline src/components/Phone.js]: [REQUIRED] Whenever I click on the icon on the details page it takes me to an undefined page. I don't think this icon or the link is needed for the details page.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @hopeforkoami, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linter are passing
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 7]
```diff
@@ -1,2 +1,219 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+- [Live Demo Link](http://pdm.nogdevhouse.com/)
+- [Demo video](https://pdm.nogdevhouse.com/capstonevideo.mp4)
```
Comment (fobadara) [inline README.md]: [REQUIRED] Good job fixing the video but you did not highlight a piece of code that you are impressed with. Also kindly ensure that your video reflects the current changes. you ade to the site.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @hopeforkoami, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
✅ Linters are passing
✅ Implemented reviewer suggestions

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @hopeforkoami 😊, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper description ✔️
- The project has met all the technical aspects you needed for this project. Keep rocking!  ✔️


Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] Kindly fix the broken MIT license link and add your own license.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**